oak_remote_attestation_noninteractive/proto/v1/messages.proto

```diff
@@ -24,10 +24,23 @@ option java_package = "oak.session.noninteractive.v1";
 message PublicKeyInfo {
   // The serialized enclave public key.
   // TODO(#3442): Specify key format.
-  bytes public_key = 1;
+  bytes enclave_public_key = 1;
   // The serialized attestation report binding the public key.
   bytes attestation = 2;
-  // TODO(#3442): Add evidence (endorsements and inclusion proofs) for the attested binary.
+  // The serialized AMD-SEV-SNP certificate.
+  bytes tee_certificate = 3;
```

Comment:
This is actually three different certificates packaged in two different files for AMD SEV-SNP.

Reply:
I'll add this to the comment. Do you have a reference I could include in the comment?

Perhaps I could define a separate struct for it here. WDYT?

Reply:
The information specific to AMD SEV-SNP is in https://www.amd.com/system/files/TechDocs/57230.pdf.

I don't think this message will only be used form AMD SEV-SNP, so perhaps the field can be renamed to indicate that it contains multiple certificates and add a comment that the details of how the certificates are encoded in this byte array are implementation-specific.